rpcs3/Crypto/unedat.h

```diff
@@ -6,6 +6,10 @@
 
 #include "Utilities/File.h"
 
+#ifdef _WIN32
+#undef stat
+#endif
```

Comment:
why

Reply:
It doesn't compile without the change.
```
In file included from C:/src/rpcs3/3rdparty/wolfssl/wolfssl/wolfssl/wolfcrypt/wc_port.h:687,
                 from C:/src/rpcs3/3rdparty/wolfssl/wolfssl/wolfssl/wolfcrypt/types.h:35,
                 from C:/src/rpcs3/3rdparty/wolfssl/wolfssl/wolfssl/wolfcrypt/asn_public.h:34,
                 from C:/src/rpcs3/3rdparty/wolfssl/wolfssl/wolfssl/ssl.h:35,
                 from C:/src/rpcs3/rpcs3/Emu/NP/rpcn_client.h:36,
                 from C:/src/rpcs3/rpcs3/rpcs3qt/rpcn_settings_dialog.h:9,
                 from C:/src/rpcs3/rpcs3/rpcs3qt/main_window.cpp:13:
C:/src/rpcs3/rpcs3/Crypto/unedat.h:98:20: error: 'fs::stat_t EDATADecrypter::_stat64()' marked 'override', but does not override
   98 |         fs::stat_t stat() override
      |                    ^~~~
C:/src/rpcs3/rpcs3/Crypto/unedat.h: In member function 'fs::stat_t EDATADecrypter::_stat64()':
C:/src/rpcs3/rpcs3/Crypto/unedat.h:100:47: error: 'class fs::file' has no member named '_stat64'; did you mean 'stat'?
  100 |                 fs::stat_t stats = edata_file.stat();
      |                                               ^~~~
```

Reply:
that makes no sense. but undefining stuff is never the answer.
You'll have to find the proper cause of this

Reply:
rename fs::file::state to get_stat.

Reply:
Renamed.